src/NearOrg/Notifications/NotificationsList.jsx

```diff
@@ -25,236 +18,22 @@ const renderItem = (item, i) => {
   return (
     <Widget
       src="${REPL_ACCOUNT}/widget/NearOrg.Notifications.Notification"
-      key={i}
+      key={JSON.stringify(item)}
```

Comment:
Depending on the size of the `item` object, this could potentially create a performance issue. Is there a value or two inside of `item` that we can combine to come up with a unique key instead of stringifying the entire object?

Reply:
Yes. There are `accountId`, `blockHeight`. I think we can combine `blockHeight` + `index`